modular-docs-manual/content/topics/creating_procedure_modules.adoc

```diff
@@ -1,13 +1,41 @@
+.Procedure introduction
+The procedure introduction is a short paragraph that provides an overview of the module. The introduction includes what the module will help the user do and why it will be beneficial to the user. It includes key words that relate to the module to maximize search engine optimization in the first paragraph of the introduction.
 
-* Download the link:https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc[procedure module template (adoc file)] for new projects.
-* For real-world examples of procedure modules, see <<modular-docs-procedure-examples>>.
+The introduction typically provides context for the procedure, such as:
+
+* Why and where the user performs the procedure
+* Special considerations specific to the procedure
+
+.Procedure prerequisites
+This section is optional. Prerequisites are a bulleted list of conditions that must be satisfied before the user starts the procedure. Use a bulleted list for prerequisites and the plural heading *Prerequisites*, even if you only have a single prerequisite.
+
+Prerequisites can be full sentences or sentence fragments, however prerequisite list items are parallel.
+
+Focus on relevant prerequisites that users might not otherwise be aware of. Do not list obvious prerequisites. If a prerequisite applies to all of the procedures in a user story, consider listing the prerequisite in the assembly file. If you do this, consider including the prerequisite in the procedure module as a comment.
+
+.Procedure steps
+The procedure consists of one or more steps required to complete the procedure. Each step describes one action. For single-step procedures, use an unnumbered bullet instead of a numbered list.
+
+NOTE: Not all numbered lists in documentation are procedures. You can also use numbered lists in any module type for non-procedural sequences, such as a process flow of system actions.
+
+.Procedure verification
+This section is optional. It provides the user with one or more steps to verify that the procedure provided the intended outcome, for example:
+
+* An example of expected command output or a pop-up window that the user receives when the procedure is successful
+* Actions for the user to complete, such as running a command, to determine the success or failure of the procedure
+
+.Procedure additional resources
+This section is optional. The additional resources list links to other material closely related to the contents of the procedure module, such as other documentation resources, instructional videos, or labs.
+
+Focus on relevant resources that are likely to be of immediate interest to the user. Do not list resources for completeness. If a resource applies to all of the modules in a user story, consider listing the resource in the Additional resources section of the assembly file. If you do this, consider including the resource in the procedure module as a comment.
```

Comment:
Can be briefer IMHO:
"are likely to be of immediate interest" -> might interest